<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>艺画文心</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pi/fuck_test_2.png);
color: rgb(255, 255, 255);
background-color: #970804;
font: 18pt &quot;楷体&quot;;

</string>
    </property>
    <property name="text">
     <string>艺画文心</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>64</y>
      <width>164</width>
      <height>536</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView {
	
	background-color: rgb(88, 15, 5);
	color: rgb(255, 255, 255);
}
QListView::item{
background-color: transparent;
height:40px;
padding-left:4px;

}

QListView::item:hover {
	background-color:  transparent;
	color: #dfbe96;

}
QListView::item:selected {
	/*background-color: transparent;*/
background-color:  transparent;
color: #dfbe96;

border-left: 2px solid #ffffff;
}
QListWidget::item:hover {
    border-left: 2px solid white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>丹青流转</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources/img.qrc">
       <normaloff>:/icon/魔方_cube-five.svg</normaloff>:/icon/魔方_cube-five.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>墨韵生花</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources/img.qrc">
       <normaloff>:/icon/三菱_mitsubishi.svg</normaloff>:/icon/三菱_mitsubishi.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>彩墨婆娑</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources/img.qrc">
       <normaloff>:/icon/六边立体_hexagonal.svg</normaloff>:/icon/六边立体_hexagonal.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>介绍和帮助</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources/img.qrc">
       <normaloff>:/icon/三棱锥_triangular-pyramid.svg</normaloff>:/icon/三棱锥_triangular-pyramid.svg</iconset>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>64</y>
      <width>836</width>
      <height>536</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="button_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>410</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
}

QPushButton:pressed {
    background-color: #a0a0a0;
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>下载</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_1">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
}

QPushButton:pressed {
    background-color: #a0a0a0;
    border-style: inset;
}
</string>
      </property>
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
     <widget class="QLabel" name="imglabel_1_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>330</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
</string>
      </property>
      <property name="text">
       <string>生成图片</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="imglabel_1_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>330</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;</string>
      </property>
      <property name="text">
       <string>拖拽放入图片</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/fuck_test_1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>836</width>
        <height>536</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/pi/ljz_2.png);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/微信图片_20230412225934.png);
font: 14pt &quot;楷体&quot;;
color: rgb(223, 190, 150);</string>
      </property>
      <property name="text">
       <string>丹青流转</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>listView</zorder>
     <zorder>button_2</zorder>
     <zorder>button_1</zorder>
     <zorder>imglabel_1_2</zorder>
     <zorder>imglabel_1_1</zorder>
     <zorder>label_11</zorder>
     <zorder>label_3</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListView" name="listView_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/pi/ljz_2.png);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
}

QPushButton:pressed {
    background-color: #a0a0a0;
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
     <widget class="QLabel" name="imglabel_2_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>330</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
</string>
      </property>
      <property name="text">
       <string>生成图片</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_14">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>410</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
}

QPushButton:pressed {
    background-color: #a0a0a0;
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>下载</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>311</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/微信图片_20230412225930.png);
font: 14pt &quot;楷体&quot;;
color: rgb(223, 190, 150);</string>
      </property>
      <property name="text">
       <string>墨韵生花</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/背景2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>330</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;</string>
      </property>
     </widget>
     <zorder>listView_5</zorder>
     <zorder>button_13</zorder>
     <zorder>imglabel_2_2</zorder>
     <zorder>button_14</zorder>
     <zorder>label_5</zorder>
     <zorder>textEdit</zorder>
     <zorder>label_28</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="button_19">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
}

QPushButton:pressed {
    background-color: #a0a0a0;
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
     <widget class="QLabel" name="imglabel__3_1">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>330</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
</string>
      </property>
      <property name="text">
       <string>生成图片</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_20">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>410</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;
}

QPushButton:pressed {
    background-color: #a0a0a0;
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>下载</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/微信图片_20230412225934.png);
font: 14pt &quot;楷体&quot;;
color: rgb(223, 190, 150);</string>
      </property>
      <property name="text">
       <string>彩墨婆娑</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="imglabel_3_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>330</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;</string>
      </property>
      <property name="text">
       <string>拖拽放入图片</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>161</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/背景3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListView" name="listView_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>836</width>
        <height>536</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/pi/ljz_2.png);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <zorder>listView_10</zorder>
     <zorder>imglabel__3_1</zorder>
     <zorder>button_19</zorder>
     <zorder>button_20</zorder>
     <zorder>imglabel_3_1</zorder>
     <zorder>label_8</zorder>
     <zorder>label_33</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QListView" name="listView_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>836</width>
        <height>536</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/pi/ljz_2.png);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>521</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>丹青流转：基于 CycleGAN等循环生成对抗模型的中国山水画的山水画皱法风格迁移模块</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>521</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>墨韵生花：基于Stable Diffusion 扩散模型的文字生成中国山水画模块</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>661</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>彩墨婆娑：基于Stable Diffusion 扩散模型的中国山水画图像上色模块以及基于卷积神经网络的图像风格转换模块</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>551</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #ffffff;
    color: #000000;
    border: 1px solid #000000;
    border-radius: 10px;
    padding: 5px;
    font-family: &quot;微软雅黑&quot;;
    box-shadow: 2px 2px 4px #888888;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;    欢迎大家来到本团队开发的“艺画文心——中国传统艺术绘画创作系统”，为了传承发扬中国传统，让更多人爱上山水画，本团队开发这款软件。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;    使用“丹青流转”、“彩墨婆娑”的时候，需要将待处理的图片的拖住放入左边的框中，点击运行按钮，等待一段一分钟左右的处理时间后，生成的图片会显示在右边的框中，点击下载按钮即可保存生成的图片。  &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;      使用“墨韵生花”时，需要在左边的文本输入框，输入您想要生成的图片的描述，点击运行，等待一段时间，生成的图片将会显示在右边，点击下载按钮即可保存&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>60</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/微信图片_20230412225924.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>470</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pi/微信图片_20230412225900.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>0</y>
      <width>836</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #970804;
font: 16pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>中国传统艺术绘画AI创作系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources/img.qrc"/>
 </resources>
 <connections/>
</ui>
